webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -260,6 +275,14 @@ export default {
     categoryIds(categories) {
       return categories.map(c => c.id)
     },
+    deleteImage() {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-16T14:45:15Z" title="Thursday, January 16th 2020, 3:45:15 pm +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -326,4 +339,11 @@ export default {
     }
   }
 }
+.delete-image {
+  right: 3px;
+  position: relative;
+  z-index: 1;
+  float: right;
+  top: 42px;
```


can we use tokens for these values instead of magic numbers?